GECKOInstaller.m

```diff
@@ -0,0 +1,43 @@
+classdef GECKOInstaller
+% GECKOInstaller
+%   Support for installing and uninstalling
+%   Run GECKOInstaller.install to install (will set up the path in MATLAB)
+%   Run GECKOInstaller.uninstall to clear the path from MATLAB
+%   To install, you first need to cd to the GECKO root.
+%
+% Johan Gustafsson, 2022-07-07
+%
+    methods (Static)
+        function install
+            sourceDir = fileparts(which(mfilename));
+            paths = GECKOInstaller.GetFilteredSubPaths(sourceDir, '.*\.git.*');
```

Comment:
There is some duplicated code with regards to path definition (`.*\.git.*`), it would be much nicer to have paths defined only once.

Reply:
Fixed